JS/lesson-16/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+// let admin;
+// let userName = 'Jon';
+// admin = userName;
+// alert(admin);
+
+// let ourPlanet;
+// let userName;
+
+// let x = 20;
+// let y = 58;
+// let z = 42;
+// let rezult = x+y+z;
+// console.log(rezult);
+
+// let sec = 60;
+// let minute = 60;
+// let hour = 24;
+// let day = 365;
```

Comment:
так как эти значение неизменны, то стоит использовать `const`
```suggestion
// const SECONDS_IN_MINUTE = 60;
// const MINUTES_IN_HOUR = 60;
// const HOURS_IN_DAY = 24;
// const DAYS_IN_YEAR = 365;
```